source/base/quadrature.cc

```diff
@@ -210,7 +210,10 @@ Quadrature<0>::Quadrature(const Quadrature<1> &)
   // quadrature_points(1),
```

Comment:
It is somewhat strange that `quadrature_points(1)` has been commented out here. It makes points and weights different in size, but it's clearly intentional.